src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -328,7 +328,7 @@ Modelers:
     <value>Parameters "subscriptionId" and "api-version" are not allowed in the operations section</value>
   </data>
   <data name="ServiceDefinitionParametersMissingMessage" xml:space="preserve">
-    <value>Parameters "subscriptionId" or "api-version" are referenced but not defined in the parameters section of Service Definition</value>
+    <value>Parameters "subscriptionId" and "api-version" must be defined in the parameters section in the swagger for the Azure Management Plane service.</value>
```

Comment:
1. Update this message
2. There is a different rule that flags a case where subscriptionId and ApiVersion are not defined in the global parameters section, you might want to modify the message to reflect that these parameters are not expected in the operations' parameters section